rbac/management/migrations/0070_resourcedef_workspace_join_table.py

```diff
@@ -0,0 +1,105 @@
+import uuid
+
+from django.db import migrations, models
+from django.db.backends.postgresql.schema import DatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+from management.role.model import ResourceDefinitionsWorkspaces
+
+
+def link_resource_definitions_workspaces(apps: StateApps, schema_editor: DatabaseSchemaEditor):
```

Comment:
Should we make it an internal endpoint we can call? I am wondering if there is any situation that we wanna run it again. For example, some workspace missing that we import later. or in fedramp env, this migration got run before the workspace get imported, so we have to re-run this